src/main/java/org/folio/circulation/domain/RequestQueue.java

```diff
@@ -28,6 +28,11 @@ Request getHighestPriorityFulfillableRequest() {
     return fulfillableRequests().get(0);
   }
 
+  public Request getLowestPriorityFulfillableRequest() {
+    return fulfillableRequests().get(fulfillableRequests().size() - 1);
```

Comment:
What if there are no fulfillable requests in the queue?

Reply:
Yes, I added this check afterward and it will return null.

Reply:
This code isn't needed anymore. I removed it already.